cloudify_rest_client/labels.py

```diff
@@ -0,0 +1,45 @@
+from cloudify_rest_client.responses import ListResponse
+
+
+class Label(dict):
+    def __init__(self, label):
+        super(Label, self).__init__()
+        self.update(label)
+
+    @property
+    def key(self):
+        return self['key']
+
+    @property
+    def value(self):
+        return self['value']
+
+    @property
+    def created_at(self):
+        return self['created_at']
+
+    @property
+    def creator_id(self):
+        return self['creator_id']
+
+
+class DeploymentsLabelsClient(object):
+    def __init__(self, api):
+        self.api = api
+
+    def list_keys(self):
+        """
+        Retunrns a list of the deployments labels' keys in the current tenant.
```

Comment:
```suggestion
        Returns a list of the deployments labels' keys in the current tenant.
```

Reply:
And tbh, you know that I like to drop as many words as I can get away with!

```suggestion
    All defined label keys, from all deployments.
```
Because
 - "Returns a list" is unnecessary, of course a method named "list keys" is going to return a list
 - "list" is doubly unnecessary, because what if it's not a list but a set? A set works even better. And btw, it's not even a list, it's a ListResponse
 - "in the current" tenant: all the restclient methods work in terms of the current tenant (modulo exceptions, with an explicit "all tenants", but still: current tenant is implicit even then), so no need to say that just here specifically

So we would maybe just drop the whole docstring then? But I feel it's important to underline that yes, that does look at all deployments